scylla-cdc-replicator/src/main.rs

```diff
@@ -1,29 +1,58 @@
 mod replication_tests;
 pub mod replicator_consumer;
 
-use replicator_consumer::ReplicatorConsumerFactory;
-use scylla::SessionBuilder;
-use std::error::Error;
-use std::sync::Arc;
 use std::time::Duration;
 
+use clap::Parser;
+
+use replicator_consumer::Replicator;
+
+#[derive(Parser)]
+struct Args {
+    #[clap(short, long)]
+    keyspace: String,
+
+    #[clap(short, long)]
+    table: String,
+
+    #[clap(short, long)]
+    source: String,
+
+    #[clap(short, long)]
+    destination: String,
+
+    #[clap(long, default_value_t = 60)]
+    window_size: u16,
+
+    #[clap(long, default_value_t = 30)]
+    safety_interval: u16,
+
+    #[clap(long, default_value_t = 10)]
+    sleep_interval: u16,
```

Comment:
Please make it more clear what are the units here. Moreover, if it's seconds then I think a `double` would be a better choice so that sub-second intervals can be specified.

Maybe `clap` allows to just parse `Duration`s?

Reply:
Not `Duration`s per se but Duration wrapped in custom type with some traits implemented.
This of course requires parsing command line inputs into durations. Do you have any specific format in mind? Just a finite decimal representation of a number of seconds?

Reply:
Hmm, no, I didn't have any specific format in mind. I thought that maybe Rust's standard library makes it possible to parse durations in some format, but if not then let's stick to `f64` and interpret as seconds. No need to write any custom wrappers.